index.js

```diff
@@ -99,12 +99,32 @@ module.exports = {
 
     // We shouldn't enforce filename casing to always be the same.  e.g. We may
     // prefer kebab-case for bin files, PascalCase for React components, etc.
+    // https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/filename-case.md
     'unicorn/filename-case': 'off',
 
     // Uses safe-regex to disallow potentially catastrophic exponential-time
     // regular expressions.
+    // https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/no-unsafe-regex.md
     'unicorn/no-unsafe-regex': 'error',
 
+    // This rule is onerous in React functional components, has been debated
+    // by the community and proven as an unnecessary optimization
+    // https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/consistent-function-scoping.md
+    'unicorn/consistent-function-scoping': 'off',
+
+    // This rule is incompatible with React
+    // https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/no-null.md
+    'unicorn/no-null': 'off',
+
+    // Preventing abbreviations is too opinionated for linting and is not
+    // particularly compatible with React development (props, ref, etc.)
+    // https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/prevent-abbreviations.md
+    'unicorn/prevent-abbreviations': 'off',
+
+    // This rule is overly burdensome and is premature optimization, at best
```

Comment:
>This rule is overly burdensome

doesn't seem to convey the concern as clearly as it could

Reply:
Just a side note. It's frustrating that these comments do not include the rule you're discussing, just the comment above the rule. Could you please include the line of code that is actually in question so people don't have to open the file and find the line in a separate window?

This is a bad rule on its face. It isn't deserving of any deeper discussion. I'd rather not write any comment at all. My "at best" was me being nice.

I didn't decide to include the unicorn ruleset. It's is a very opinionated rule set. I'd rather just group them all under a single comment of not agreeing with all of the strong opinions that this ruleset has.